.github/workflows/auto-approve.yaml

```diff
@@ -17,14 +17,14 @@ name: auto-approve
 jobs:
   approve:
     runs-on: ubuntu-latest
-    if: contains(github.head_ref, 'autosynth')
+    if: contains(github.head_ref, 'yoshi-code-bot')
```

Comment:
head-ref is the branch name

Reply:
Branch names for PRs from yoshi-code-bot is `yoshi-code-bot:regenerate-securitycenter`